r/src/R/ai/h2o/sparkling/SharedBackendConf.R

```diff
@@ -111,7 +111,10 @@ SharedBackendConf <- setRefClass("SharedBackendConf", methods = list(
 
     clientNetworkMask = function() { ConfUtils.getOption(invoke(jconf, "clientNetworkMask")) },
 
-    ignoreSparkPublicDNS = function() { invoke(jconf, "ignoreSparkPublicDNS") },
+    ignoreSparkPublicDNS = function() {
+      print("Method 'ignoreSparkPublicDNS' is deprecated in release 3.30 without replacement.")
```

Comment:
I'm not sure how to generate warnings in R properly, but it seems that there is a method for that https://stat.ethz.ch/R-manual/R-devel/library/base/html/warning.html